examples/gno.land/p/sys/users/users.gno

```diff
@@ -0,0 +1,43 @@
+package users
+
+import (
+	"std"
+)
+
+type UserLookup struct {
+	Name  string
+	Addr  std.Address
+	Alias bool // lookup was found via an alias
+}
+
+// NewUserLookup returns a new UserLookup struct
+func NewUserLookup(name string, addr std.Address, alias bool) UserLookup {
+	return UserLookup{
+		Name:  name,
+		Addr:  addr,
+		Alias: alias,
+	}
+}
+
+func (lookup UserLookup) Exists() bool {
+	return lookup.Addr != ""
+}
+
+func (lookup UserLookup) Deleted() bool {
+	return lookup.Addr == "" && lookup.Name == "ghost" // todo see "ghost"?
+}
+
+// AddressOrName can be used for resolving either the address or the username
+type AddressOrName string
```

Comment:
do we still need this?

Reply:
rm'd [c455438](https://github.com/gnolang/gno/pull/3166/commits/c455438d1f1e5287dff66683e64261fff4715e98)